tests/phpunit/tests/link/getEditCommentLink.php

```diff
@@ -0,0 +1,93 @@
+<?php
+/**
+ * @group link
+ * @group comment
+ * @covers ::get_next_comments_link
+ */
+class Tests_Link_GetEditCommentLink extends WP_UnitTestCase {
+
+	public static $comment_ids;
+	public static $user_ids;
+
+	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
+		self::$comment_ids = array(
+			'valid'   => $factory->comment->create( array( 'comment_content' => 'Test comment' ) ),
+			'invalid' => 12345, // Invalid comment ID
+		);
+
+		self::$user_ids = array(
+			'admin'      => $factory->user->create( array( 'role' => 'administrator' ) ),
+			'subscriber' => $factory->user->create( array( 'role' => 'subscriber' ) ),
+		);
+	}
+
+	public function set_up() {
+		parent::set_up();
+		wp_set_current_user( self::$user_ids['admin'] );
+	}
+
+	public function test_get_edit_comment_link_display_context() {
```

Comment:
```suggestion
	/*
	 * @ticket 61727
	 */
	public function test_get_edit_comment_link_display_context() {
```

Add `@ticket` annotation to other tests also.